Exercise_F.js

```diff
@@ -10,6 +10,7 @@
 
 function occurrence(string, targetChar) {
   // CODE HERE
+  return string.split(targetChar).length - 1
 }
 

```

Comment:
This is a very smart way of getting the number of characters within. Fantastic work!